doc/user/content/concepts/views.md

```diff
@@ -27,10 +27,10 @@ All views in Materialize are built by reading data from
 
 ## Non-materialized views
 
-A non-materialized view saves a query under a name to provide a shorthand for
-referencing the query. The query is **not** executed during non-materialized
-view creation, and a non-materialized view does **not** persist its results in
-durable storage.
+A non-materialized view, also generally referred to as a "view", saves a query
```

Comment:
So, this is a little bit opposite of what we had discussed.  But, changing mentions of non-materialized views to views in this section required a bit more finessing of the sentences than I had thought.  So, will punt on that now.

Reply:
Also makes sense! Just important to have the discussion and clarify you should feel empowered to obliterate these terms.